config/initializers/sidekiq.rb

```diff
@@ -4,6 +4,8 @@
 
 Sidekiq.configure_server do |config|
   config.redis = REDIS_CONFIG['redis']
+  config.super_fetch! if Rails.env.production?
```

Comment:
```suggestion
  config.super_fetch! unless Settings&.sidekiq&.disable_super_fetch
```

I think'd it'd be best to enable this for local environments by default, and then have a mechanism to disable it if a developer wishes to do so.

Reply:
I really don't like the &.&.&. pattern -- its an error if we expect a setting and its not there

Reply:
I just remembered that [we already have the `EXCLUDE_SIDEKIQ_ENTERPRISE` env var](https://github.com/department-of-veterans-affairs/vets-api/blob/master/Gemfile#L173), so maybe we could re-write to:

```ruby
config.super_fetch! unless ENV['EXCLUDE_SIDEKIQ_ENTERPRISE'] == 'true'
```